index.js

```diff
@@ -108,7 +109,8 @@ async function smartContractInitialization() {
     log("Minting 1000000 tokens to following addresses:")
     for (const address of privateKeys.map(computeAddress)) {
         log("    " + address)
-        await token.mint(address, "1000000")
+        const mintTx = await token.mint(address, "1000000")
+        await mintTx.wait()
     }
```

Comment:
this is old code, but I saw rejection issue because it submits before waiting for previous tx to finish. this fix ensures it waits for prev tx to finish